test/dest.js

```diff
@@ -763,53 +763,6 @@ describe('dest stream', function() {
     stream.end();
   });
 
-  it('should write file atime and mtime using the vinyl stat', function(done) {
```

Comment:
> Ah. Yeah, those things seem very wrong :P

I thought so. And yet all of this passes the tests...

Effectively, I guess, the current behaviour of the getter and setter of `contents` is correct from the pov of a plugin, but we must kind of invert things at the terminals `src` and `dest`?

When `src` sets `file.contents` this represents an access of the physical source file it reflects. So the setter of `contents`, as called by `src`, should update `atime`. The `mtime` should be the same as the source file.

When `dest` gets `file.contents` this represents a modification of the file on disk, and so it should update `mtime`. The `atime` of the physical destination file should be the same is the vinyl.

Does that make any sense?